scripts/enum/mob_skills.lua

```diff
@@ -35,13 +35,22 @@ xi.mobSkill =
 
     SILENCE_GAS_1            =  314,
     DARK_SPORE_1             =  315,
+    DRILL_BRANCH             =  328,
+    PINECONE_BOMB            =  329,
+    LEAFSTORM                =  331,
```

Comment:
And between this 2 aswell. And down bellow theres a few other cases.

 If the enums dont follow a direct progression, separate them